apollo-converters/moshi/src/test/java/com/apollostack/android/HeroDetailsDataPOJO.java

```diff
@@ -0,0 +1,41 @@
+package com.apollostack.android;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+class HeroDetailsDataPOJO implements HeroDetails.Data {
```

Comment:
haha yeah it's pretty mechanical and boring, but I think this should be opt-in. That is, we still generate the interfaces as usual and if you want, we can also generate POJOs or otherwise you can use your own stuff like AutoValue for example